aten/src/ATen/cpu/vec256/intrinsics.h

```diff
@@ -32,6 +32,20 @@
         (defined(__VEC__) || defined(__ALTIVEC__))
 /* XLC or GCC-compatible compiler, targeting PowerPC with VMX/VSX */
 #include <altivec.h>
+using __vd = __vector double;
```

Comment:
Please do not introduce names with leading double-underscore. That is undefined behavior in C++

Reply:
ok